pkg/mover/chart.go

```diff
@@ -236,17 +285,116 @@ func namespacedPath(fullpath, chartName string) string {
 }
 
 /*
-Move executes the Chart relocation which includes
+  Move performs the relocation.
+
+A regular move executes the Chart relocation which includes
 
 1 - Push all the images to their new locations
 
 2 - Rewrite the Helm Chart and its subcharts
 
 3 - Repackage the Helm chart as toChartFilename
+
+A save to an offline tarball bundle will:
+
+1 - Drop all images to disk, with the original chart (unpacked) and hints file
+
+2 - Package all in a single compressed tarball
 */
 func (cm *ChartMover) Move() error {
+	if cm.targetOfflineTar != "" {
+		return cm.saveOfflineBundle()
+	}
+	return cm.moveChart()
+}
+
+func (cm *ChartMover) saveOfflineBundle() error {
```

Comment:
Could we add some tests for this method?

Reply:
What about moving these methods to another file in the same package? something like offlinebundle.go?

Reply:
I think I can do both in another PR. I was already thinking about moving all this to its own file.

Reply:
ok, I just wanted to close this. Seems we keep on adding...

Reply:
I am moving this to a separate source file in this diff, but the testing will be added in the second step. It will be easier as I have both save and load I can just verify a load and save produces the original test bundle.

Reply:
Also we are disabling test we already had anyways so...